deployment/config.go

```diff
@@ -18,19 +18,21 @@ import (
 // Config contains the necessary data
 // to deploy and provision a load test environment.
 type Config struct {
-	ClusterName         string // Name of the cluster.
-	AppInstanceCount    int    // Number of application instances.
-	AppInstanceType     string // Type of the EC2 instance for app.
-	AgentInstanceCount  int    // Number of agents, first agent and coordinator will share the same instance.
-	AgentInstanceType   string // Type of the EC2 instance for agent.
-	EnableAgentFullLogs bool   // Logs the command output (stdout & stderr) to home directory.
-	ProxyInstanceType   string // Type of the EC2 instance for proxy.
-	SSHPublicKey        string // Path to the SSH public key.
-	DBInstanceCount     int    // Number of DB instances.
-	DBInstanceType      string // Type of the DB instance.
-	DBInstanceEngine    string // Type of the DB instance - postgres or mysql.
-	DBUserName          string // Username to connect to the DB.
-	DBPassword          string // Password to connect to the DB.
+	ClusterName           string // Name of the cluster.
+	VpcID                 string // Id of the VPC to be used
+	AppInstanceCount      int    // Number of application instances.
+	AppInstanceType       string // Type of the EC2 instance for app.
+	AgentInstanceCount    int    // Number of agents, first agent and coordinator will share the same instance.
+	AgentInstanceType     string // Type of the EC2 instance for agent.
+	ElasticSearchSettings ElasticSearchSettings
```

Comment:
Interesting approach, we were using a flat order here. I think it would be better to keep it consistent. Either group `app`, `agent`, `db` etc. or remove the ES struct here and add individual configurations here. 0/5 on the both TBH.

Reply:
Hum, we have the `LoggerSettings`. I think we should have the configuration grouped as we have in the `config.json`. I'd leave it as it is because all the configuration related code is based on that :\. But if @agnivade is agree with you I'll change it

Reply:
I am fine with grouping. Doesn't make too much of a difference either ways. Let's go with this for now.